.testhelpers.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
rename to `.helpers.sh` or something else more indicative that this is not a file for testing FATS.

Reply:
done.